src/pages/java/loops/continue-control-statement/index.md

```diff
@@ -3,7 +3,7 @@ title: Continue Control Statement
 ---
 # Continue Control Statement
 
-The `continue` statement makes a loop skip all the following lines after the continue and jump ahead to the beginning of the next iteration. In a `for` loop, control jumps to the update statement, and in a `while` or `do while` loop, control jumps to the boolean expression/condition.
+The `continue` statement makes the loop skip all the statements following after the `continue` statement and jump ahead to the beginning of the next iteration. In a `for` loop, control jumps to the update statement; and in a `while` or `do while` loop, control jumps to the boolean expression/condition.
```

Comment:
The change to the first sentence does not add anything, and the rewording doesn't really make it clearer than what was here. The change to a semi-colon is also not correct here.

Reply:
The rewording makes the text more technical, by referring to code as statements and not just "lines". I figured the guide maintainers would be interested in accurate wording and grammar.

Also, semicolons are used to link two independent clauses of equal rank or position. I linked them here to give the reader an immediate contrast in the execution of 'continue' for the different kinds of loops.

Reply:
I see you're right about the semicolon use in a list that contains commas, which I think fits what you've done. "Code" and "statements" are not synonymous though, as there are expressions that are also skipped.